### ISSUE TITLE
where is pretrained model

### ISSUE THREAD

**Issue by xd1073321804** [monikaBeizi/CNN-face-emotion-recognition#9]
can you provide the pretrained model?

**Comment by monikaBeizi** [comment]
I have a model for the FER dataset, but I haven’t finished the one for FERPlus yet. I can provide you with a model for training the FER dataset.
[model_for_fer.zip](https://github.com/monikaBeizi/CNN-face-emotion-recognition/files/13667609/model_for_fer.zip)

**Comment by monikaBeizi** [comment]
The FerPlus model is now completed. You can find the model in the 'models' folder of the 'ez_vision' branch, inside vgg.py. It achieves an accuracy of 0.84 on the test set.
